sklearn/utils/validation.py

```diff
@@ -342,17 +342,19 @@ def check_memory(memory):
     Parameters
     ----------
     memory : None, str or object with the joblib.Memory interface
+        If string or None, the location on which we want to create the joblib.Memory \
+            interface.
```

Comment:
```suggestion
        If string, the location on which we want to create the `joblib.Memory`
        interface. If None is given, no caching is done and the Memory object is
        completely transparent.
```

Otherwise LGTM

Reply:
I would even do
```suggestion
        - If string, the location where to create the `joblib.Memory` interface.
        - If None, no caching is done and the Memory object is completely transparent.
```

Reply:
I would even do
```suggestion
        - If string, the location where to create the `joblib.Memory` interface.
        - If None, no caching is done and the Memory object is completely transparent.
```